src/pages/productDetailsPage/ProductDetails.jsx

```diff
@@ -85,6 +91,34 @@ const ProductDetails = () => {
   // current Object ID from URL
   const currentObjectID = location.pathname.split('/')[3];
 
+  const [cart, setCart] = useRecoilState(cartState);
+  const [removed, setRemoved] = useRecoilState(removedItem);
+
+  const addToCart = (it) => {
```

Comment:
feels like these needs more comments, not exactly clear to me what's going on

Reply:
remove because it was old function it's all centralised in a selector now in recoil

Reply:
> Have you checked if this can be lazy loaded and how it affects the bundle size?
> 
> Overall, really well done, Alex! Left some small comments, mostly cleanup and naming.

No I didn't realize that

Reply:
You can watch centralised function : 
`src/config/cartFunctions.js`